tox.ini

```diff
@@ -0,0 +1,45 @@
+[tox]
+envlist = py{27,37}, docs
+skip_missing_interpreters = true
+
+
+[testenv]
+passenv = SSL_CERT_FILE CI TRAVIS* APPVEYOR* CODECOV*
+setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+usedevelop = True
+deps = -rrequirements.txt
+       -rtest/requirements.txt
+commands = pytest {posargs:-vv --cov-report term-missing --cov=sureal/}
+
+[testenv:coverage]
+passenv = {[testenv]passenv}
+setenv = COVERAGE_FILE={toxworkdir}/.coverage
+basepython = python
+deps = coverage
+commands = coverage combine
+           coverage report -i -m
+           coverage xml -i -o "{toxworkdir}/coverage.xml"
+           coverage html -i -d "{toxworkdir}/coverage"
+
+[testenv:docs]
+passenv = {[testenv]passenv}
+basepython = python
+deps = readme_renderer
+commands = python setup.py check --strict --restructuredtext
+           python setup.py explain -c125
+
+[testenv:venv]
+basepython = python3
```

Comment:
This is a convenient way of quickly getting a virtualenv for local iteration.
It drops it in a customary place (`.venv` folder) where PyCharm for example finds it automatically.

One would just run `tox -e venv` to get a ready to use local venv for development